### DISCUSSION TITLE
Theme selector issue

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11577] [Q&A]
Folks please help ob below issue: that happen automatically even i dont do the special thing:

InvalidOperationException: Unable to resolve service for type 'OrchardCore.Admin.IAdminThemeService' while attempting to activate 'OrchardCore.Users.Services.UsersThemeSelector'.

Facing below error:

![image](https://user-images.githubusercontent.com/5202291/164539345-ad5ea095-9806-42df-a62c-8914d3fb1629.png)

On browser:

![image](https://user-images.githubusercontent.com/5202291/164539450-e82b27b8-7ac1-4f4f-833f-fa9705af9166.png)

**Comment by hishamco** [comment]
Does the users module enabled?

**Reply by saleemhaq** [reply]
> Does the users module enabled?

I unable to do any activity not admin side also open it occur suddenly??

**Reply by saleemhaq** [reply]
> Does the users module enabled?

how i check in that state?

**Reply by hishamco** [reply]
Go to the admin -> Features

**Reply by saleemhaq** [reply]
> Go to the admin -> Features

I unable to login to dashboard still face this issue:

![image](https://user-images.githubusercontent.com/5202291/164549824-ac8fff55-b094-4cf8-8e15-c879b691e541.png)

please help.

**Reply by saleemhaq** [reply]
> Go to the admin -> Features

I unable to login to dashboard see:

![image](https://user-images.githubusercontent.com/5202291/164550298-545266a9-0fcf-4b2d-a70a-4a38f79cad89.png)

**Comment by ns8482e** [comment]
How did you come to this state? can you provide steps to reproduce?

**Reply by saleemhaq** [reply]
> How did you come to this state? can you provide steps to reproduce?

I dont do anythind sudden occur?
